### PR TITLE
Projects and About Section

### PR DESCRIPTION
**Pull request by PrantoshB**
- Project and About Section Added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,12 +28,140 @@
...
- <section class="bgimage"></section>
+ <div class="bgimage"></div>
+ <section class="project">
+    <h1 class="title">
+        Projects
+    </h1>
+    <div class="work_cards">
+        <img class="work_image" src="assets/Work6.png" alt="work 1">
+        <div class="workdetails">
+            <p class="work_title">Project name goes here</p>
```
Comment (christianbiring1) [inline index.html:44]: - [ ] Kindly make sure the `Project name` is grabbed inside a header tag `h2..6` for semantic reason

File: index.html [hunk 2]
```diff
@@ -27,12 +28,140 @@
...
+            </ul>
+            <button class="seeproject" type="button">
+                See this Project <i class="fa fa-arrow-right"></i>
+            </button>
+        </div>
+    </div>
+    <div class="work_cards">
+        <img class="work_image" src="assets/Work2.png" alt="work 1">
+        <div class="workdetails">
+            <p class="work_title">Project name goes here</p>
```
Comment (christianbiring1) [inline index.html:72]: - [ ] Kindly make sure the `Project name` is grabbed inside a header tag `h2..6` for semantic reason

File: index.html [hunk 3]
```diff
@@ -27,12 +28,140 @@
...
+            </ul>
+            <button class="seeproject" type="button">
+                See this Project <i class="fa fa-arrow-right"></i>
+            </button>
+        </div>
+    </div>
+    <div class="work_cards">
+        <img class="work_image" src="assets/Work3.png" alt="work 1">
+        <div class="workdetails">
+            <p class="work_title">Project name goes here</p>
```
Comment (christianbiring1) [inline index.html:86]: - [ ] Kindly make sure the `Project name` is grabbed inside a header tag `h2..6` for semantic reason

File: index.html [hunk 4]
```diff
@@ -27,12 +28,140 @@
...
+            </ul>
+            <button class="seeproject" type="button">
+                See this Project <i class="fa fa-arrow-right"></i>
+            </button>
+        </div>
+    </div>
+    <div class="work_cards">
+        <img class="work_image" src="assets/Work4.png" alt="work 1">
+        <div class="workdetails">
+            <p class="work_title">Project name goes here</p>
```
Comment (christianbiring1) [inline index.html:100]: - [ ] Kindly make sure the `Project name` is grabbed inside a header tag `h2..6` for semantic reason

File: index.html [hunk 5]
```diff
@@ -27,12 +28,140 @@
...
+                See this Project <i class="fa fa-arrow-right"></i>
+            </button>
+        </div>
+    </div>
+    
+ </section>
+ <section class="info-section">
+    <h1 class="title">
+        About
+    </h1>
```
Comment (christianbiring1) [inline index.html:130]: - [ ] Kindly make sure you use `h2..6` here and not `h1` for semantic reason

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @PrantoshB , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] Good commit message :+1: 
- [ ] Great UI :+1:  :1st_place_medal: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] In your README on the `Get started` section, kindly make sure you indicate the steps to follow to run the project locally.
`Hint: You may say:`
`git clone <project-name>`
`cd <project-name>`
`Open within an IDE`
`Open with the live server`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TuyishimireEric** [comment]
# Status: APPROVED  :+1: :+1:  
## Hi @PrantoshB  :hand: 

Thank you for Implementing the requested changes from the previous code reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All tests are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Professional README ✔️ 

Image


## Optional suggestions
- [ ]  Kindly ensure you added more PR description. It should tell us about what feature you added and what effect it has on the project. It's usually best practice to have a list of bullet point of what you implemented in the PR. 
image

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrantoshB** [comment]
I just checked that I have one h1 tag left for the "Projects" title. I'll make that one change in the next project.

So, if possible, kindly approve this PR.

Thanks.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED  :+1: :+1:  
## Hi @PrantoshB  :hand: 

Thank you for Implementing the requested changes from the previous code reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All tests are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Professional README ✔️ 

Image


## Optional suggestions
- [ ]  Kindly ensure you added more PR description. It should tell us about what feature you added and what effect it has on the project. It's usually best practice to have a list of bullet point of what you implemented in the PR. 
image

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**